tests/integration/cli/repos_test.py

```diff
@@ -61,6 +60,21 @@ def test_repos_manage(self):
         self.assertNotInSameLine(result, 'fake-repo', self.fake_override_repo_location)
         self.assertInSameLine(result, 'fake-repo', self.fake_local_repo_location)
 
+    def test_repos_manage_all_with_one(self):
+        self.run_command('repos override fake-repo {}'.format(self.fake_override_repo_location))
+        self.run_command('repos manage --all')
+        result = self.run_command('repos list')
+        self.assertNotInSameLine(result, self.fake_override_repo_location, 'fake-repo')
```

Comment:
probably worth asserting they are in the same line above the mange --all command.